File: Class_member/3.rb
# 클래스 변수
class Cs
    @@count = 0                 # 메소드 밖 클래스 안에서 최초 값을 지정
    def initialize()
        @@count = @@count + 1    # 클래스에 속한 변수 (인스턴스랑 상관 없음)
                                # 인스턴스가 생성 될 때마다 클래스 변수값을 +1
    end
    def Cs.getCount()           # 클래스 메소드
        return @@count        
    end
end

# 여기서 count 라는 변수는 클래스 변수이기 때문에 모든 인스턴스에서 동일한 값이 됨

i1 = Cs.new()
i2 = Cs.new()
i3 = Cs.new()
i4 = Cs.new()
p Cs.getCount()

File: Class_member/3.py
# 클래스 변수

class Cs(object):
    count = 0
    def __init__(self):
        Cs.count = Cs.count + 1     
        # 위에 있는 클래스 내부의 count에 접근하기 위해서는 Cs.count여야 함 
    @classmethod                # 클래스 메소드 설정하기 위해서 입력
    def getCount(cls):          # 클래스 메소드는 첫 번째 인자로 cls를 입력
        return Cs.count

i1 = Cs()
i2 = Cs()
i3 = Cs()
i4 = Cs()
print(Cs.getCount())